<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGroupPageLzy</class>
 <widget class="QWidget" name="CreateGroupPageLzy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>创建群聊界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="avatarLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>群头像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadAvatarButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>上传群头像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="invitFriendButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>邀请好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createGroupButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>创建群聊</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>149</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>群名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="groupName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>群号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="groupId"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
